Makefile

```diff
@@ -1797,16 +1799,30 @@ $(ESP_COMBINED512): $(USER1_BIN) $(USER2_BIN)
 # IMPORTANT: this only works if DISABLE_LTO is defined, e.g. `DISABLE_LTO=1 make`
 topstrings: $(PARTIAL)
 	$(Q)for f in `find . -name \*.o`; do \
-	  str=$$($(OBJDUMP) -j .rodata.str1.4 -h $$f 2>/dev/null | \
-	    egrep -o 'rodata.str1.4 [0-9a-f]+' | \
+	  str=$$($(OBJDUMP) -j .rodata.str1.1 -j .rodata.str1.4 -h $$f 2>/dev/null | \
+	    egrep -o 'rodata.str1.. [0-9a-f]+' | \
 	    awk $$(expr match "$$(awk --version)" "GNU.*" >/dev/null && echo --non-decimal-data) \
 	      -e '{printf "%d\n", ("0x" $$2);}'); \
 	  [ "$$str" ] && echo "$$str $$f"; \
 	done | \
 	sort -rn >topstrings
 	$(Q)echo "Top 20 from ./topstrings:"
 	$(Q)head -20 topstrings
-	$(Q)echo "To get details: $(OBJDUMP) -j .rodata.str1.4 -s src/FILENAME.o"
+	$(Q)echo "To get details: $(OBJDUMP) -j .rodata.str1.1 -j .rodata.str1.4 -s src/FILENAME.o"
+
+# Same as topstrings but consider all read-only data
+topreadonly: $(PARTIAL)
```

Comment:
Might be an idea - it's a real shame though, because this is a third part library that so far I've managed to use as-is. If I need to update it at a later date I'll need to re-apply these hacks (and let's face it, these hacks are going to end up *everywhere* over time).

I'm not sure if it's possible, but ideally we'd just have a list of these symbols somewhere, and would automatically force them into flash with a linker script or some other pass.